### PR TITLE
[Docs][Plus] Add Plus upgrading guide

### PR DESCRIPTION
**Pull request by CoderMaggie**
| Q               | A
| --------------- | -----
| Branch?         | 1.7
| Doc fix?        | no
| New docs?    | yes
| Related tickets | should be also updated on master after #11445 
| License         | MIT

### CODE REVIEW

File: docs/book/installation/sylius_plus_upgrading.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.. rst-class:: plus-doc
+
+Upgrading Sylius Plus
+=====================
+
+Sylius regularly releases new versions, usually every two weeks.
+Each release comes with an ``UPGRADE.md`` file, which is meant to help in the upgrading process.
+
+1. **Update the Sylius Plus version constraint by modifying the ``composer.json`` file:**
```
Comment (lchrusciel) [inline docs/book/installation/sylius_plus_upgrading.rst:9]: Can we expose more clearly, that `0.21` is just an example? 
```suggestion
For example, if one would like to update to version `0.21`:

1. **Update the Sylius Plus version constraint by modifying the ``composer.json`` file:**
```

File: docs/book/installation/sylius_plus_upgrading.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    .. code-block:: bash
+
+        composer update sylius/plus --with-all-dependencies
+
+    If this does not help, it is a matter of debugging the conflicting versions and working out how your ``composer.json``
+    should look after the upgrade.
+    You can check what version of Sylius is installed by running ``composer show sylius/plus`` command.
+
+3. **Follow the instructions found in the ``UPGRADE.md`` file for a given minor release.**
```
Comment (lchrusciel) [inline docs/book/installation/sylius_plus_upgrading.rst:29]: And add a link to UPGRADE.md of v0.21?

Comment (CoderMaggie) [reply docs/book/installation/sylius_plus_upgrading.rst:29]: not a fan of this idea, it's an example above with this version.

there is one upgrade file for all versions in this repository afaik :D

File: docs/book/installation/sylius_plus_upgrading.rst (hunk 1 above)
Comment (CoderMaggie) [reply docs/book/installation/sylius_plus_upgrading.rst:9]: it's same as in the OS upgrade guide:
https://docs.sylius.com/en/latest/book/installation/upgrading.html

**Comment by pamil** [comment]
Thank you, Magdalena! :1st_place_medal:

**Pull request merged**